android/engine/src/test/java/org/smartregister/fhircore/engine/p2p/dao/BaseP2PTransferDaoTest.kt

```diff
@@ -158,7 +159,7 @@ class BaseP2PTransferDaoTest : RobolectricTest() {
     }
     val searchQuerySlot = slot<SearchQuery>()
 
-    coVerify { fhirEngine.search<Patient>(capture(searchQuerySlot)) }
+    //    coVerify { fhirEngine.search<Patient>(capture(searchQuerySlot)) }
```

Comment:
can we remove

Reply:
Will be addressed via PR https://github.com/opensrp/fhircore/pull/2730